### PR TITLE
Adds Einstein Mercenary Equipment

### PR DESCRIPTION
**Pull request by BurpleBineapple**
### Overview

Adds two gear crates for mercenary teams (or an individual) looking to play as Einstein Engines.

The hardsuit is a carbon copy stat-wise of all other merc hardsuits, but contains an ion gun and more anti-synthetic equipment than hard lethals. The sprite is the same as the Einstein asset protection suit.

The group gear crate comes with:

x5 Einstein Voidsuits
x1 Einstein Hardsuit
x2 Energy Rifles
x1 Energy Shotgun
x4 9mm Pistols and Silencers
x8 Spare 9mm Magazines

The single gear crate comes with:

x1 Einstein Voidsuit
x1 Energy Rifle
x1 9mm Pistol and Silencer
x2 Spare 9mm Magazines

image
Picture of the "Paragon" hardsuit.

### Asset Licenses
The Paragon hardsuit sprites were made by Kyres and provided by him during a chat in the code_dungeon channel. Thanks!

### CODE REVIEW

**Comment by BurpleBineapple** [comment]
!review

**Comment by BurpleBineapple** [comment]
Not my fault it's failing this time. ZAS issues in the runtime check, again.

File: html/changelogs/purplepineapple-PR-15681.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: PurplePineapple
```
Comment (Alberyk) [inline html/changelogs/purplepineapple-PR-15681.yml:1]: Add the spriter to the author section as well.

**Review by Alberyk — DISMISSED** [review]

File: html/changelogs/purplepineapple-PR-15681.yml (hunk 1 above)
Comment (BurpleBineapple) [reply html/changelogs/purplepineapple-PR-15681.yml:1]: Oh forgot to. Thanks for the reminder

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request closed without merging**